src/frontends/onnx/frontend/src/editor.hpp

```diff
@@ -25,7 +36,7 @@ namespace onnx_editor {
 /// \note This class can be used to modify an ONNX model before it gets translated to
 ///       an ngraph::Function by the import_onnx_model function. It lets you modify the
 ///       model's input types and shapes, extract a subgraph and more.
-class ONNX_IMPORTER_API ONNXModelEditor final {
+class NGRAPH_API_DEPRECATED ONNX_IMPORTER_API ONNXModelEditor final {
```

Comment:
it's internal header. No needs to deprecate something here

Reply:
Agree that the header is internal, but the symbols are exposed in `libopenvino_onnx_frontend.so`.
`nm -gDC yourLib.so libopenvino_onnx_frontend.so | grep ONNXModelEditor`
```
0000000000149e70 T ov::onnx_editor::ONNXModelEditor::set_node_name(ov::onnx_editor::EditorNode const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
000000000014a1a0 T ov::onnx_editor::ONNXModelEditor::set_input_types(std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, ov::element::Type_t, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, ov::element::Type_t> > > const&)
000000000014b1b0 T ov::onnx_editor::ONNXModelEditor::set_tensor_name(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
...
```
In the current state, it's used in the test binary (it's a leftover after conception of making online and offline paths of ONNX FE).
@ilya-lavrenov Are exposed symbols not enough reason of need of depreciation?